### ISSUE TITLE
Feature request: Notification for when subgraph is synced

### ISSUE THREAD

**Issue by jjgonecrypto** [graphprotocol/support#35]
As the deployer of a subgraph, in order to speed up my workflow, I want a notification that the subgraph is synced. 

Browser notification or even just an email. Is that a possibility?

**Comment by ghost** [comment]
@nenadjaja A browser notification would probably be simplest here, maybe just on the subgraph detail page to start since the subscription is already open (is that right?). 

Then maybe later, on an account level with a subscription for each deployed subgraph in the background?

**Comment by nenadjaja** [comment]
Yep , we can do a simple browser notification for now. As long as people enable notifications :) @graph-carl what do you think?

**Comment by nenadjaja** [comment]
@graph-carl we decided to add a button/link to request notifications (instead of pushing them onto users). Can you please design this? 
Relates to https://github.com/graphprotocol/graph-explorer/issues/493

**Comment by hagerling** [comment]
I updated #493
<img width="974" alt="68296835-cc8ad880-0095-11ea-9b2d-0ee66601b84c" src="https://user-images.githubusercontent.com/38695152/68296913-fcd27700-0095-11ea-98ab-893de3b68ffa.png">
